### ISSUE TITLE
SyntaxError with v2.1.0 in Node v4

### ISSUE THREAD

**Issue by insin** [Urthen/case-sensitive-paths-webpack-plugin#21]
v2.1.0 was published with use of `let` but without a 'use strict' statement on the module.

Example failure in Node v4: https://travis-ci.org/insin/nwb/jobs/237641960

```
/home/travis/build/insin/nwb/node_modules/case-sensitive-paths-webpack-plugin/index.js:88

      let correctFilename = '!nonexistent';

      ^^^

SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
```

**Comment by Urthen** [comment]
Bah, sorry. I don't use v4. So just add 'use strict' and it'll be good?

**Comment by Urthen** [comment]
Was able to replicate and fix in Node 4.8.3.  Just published package version 2.1.1, try with that.

**Comment by insin** [comment]
Looks good, thanks!
